waiter/docs/how-to-waiter.md

```diff
@@ -4,7 +4,7 @@ Check out [Waiter Checklist](service-checklist.md) before you try running your s
 
 # Overview
 
-There are two ways to launch services on Waiter: the On-the-Fly API and the Token API.  The On-the-Fly API automatically starts HTTP services using custom HTTP headers coming from the client.  Alternatively, the Token API allows service maintainers to predefine a service.  Token-based services are reachable by HTTP clients using a DNS name or using a custom HTTP header, `X-Waiter-Token`.
```

Comment:
what about editing waiter/docs/parameters.md and waiter/docs/rest-api.md

Reply:
Too many to edit, so left them.

Reply:
Done now.